### ISSUE TITLE
Elsa 3 - SQL Activities Missing

### ISSUE THREAD

**Issue by KnibbsyMan** [elsa-workflows/elsa-core#4780]
Unless I’ve missed the package, there does not seem to be SQL activities like in Elsa 2. I used them to run periodic table cleansing activities and for inserting records during workflows. This for me is a really core feature missing from Elsa 3. Is there a plan to include SQL activities anytime soon? 

Furthermore, to build on the Elsa 2 SQL activities, I have a few features that would be nice on the Elsa 3 Designer side, if they are even possible:
- Select the “Name” of a connection string from a drop down of preconfigured connection strings.
- Display configured DbContexts in a drop down and use that as the connection string.
- I think Elsa 2 did this already but implement SQL syntax highlighting in the query activity.
- Support for running transactions, with monitoring for if it passed or failed as outcomes.

**Comment by sfmskywalker** [comment]
You're right, the SQL activities have yet to be ported to Elsa 3. Your additional suggestions about selecting a connection from a dropdown makes sense. If I'm not mistaken, this was already implemented for Elsa 2 using the Secrets module.
